tsconfig.base.json

```diff
@@ -77,6 +77,10 @@
       "@near-wallet-selector/coin98-wallet": [
         "packages/coin98-wallet/src/index.ts"
       ],
+      "@near-wallet-selector/xdefi": [
+        "packages/xdefi/src/index.ts"
+      ]
```

Comment:
Build is failing because of this extra parenthesis. Can you please remove it.

Reply:
good point